### PR TITLE
Fix broken links by making them absolute

### PR DESCRIPTION
**Pull request by fatihozer0**
PR closed by Hazelcast automation as no activity (>3 months). Please reopen with comments, if necessary. Thank you for using Hazelcast and your valuable contributions

### CODE REVIEW

**Comment by srknzl** [comment]
Don't know why tests did not run. Closing and opening PR to potentinally trigger it

File: api/3.12.3/docs/index.html [hunk 1]
```diff
@@ -93,7 +93,7 @@ <h3 id="hazelcast">Hazelcast</h3>
 				<pre><code class="lang-bash">docker run -p 5701:5701 hazelcast/hazelcast:3.12.6
```
Comment (srknzl) [inline api/3.12.3/docs/index.html:93]: In this file you need to fix other links. For example: https://github.com/hazelcast/hazelcast-nodejs-client/blob/be17452c2edaef1f7a49b9d379f82c06c1df78c5/api/3.12.3/docs/index.html#L198


Can you check all files for relative links?

Comment (srknzl) [reply api/3.12.3/docs/index.html:93]: You need to check all files because we might have changed the README in master in any version.

**Comment by srknzl** [comment]
@fatihozer0  It is not trivial to write a linting tool to fix html files but sure, you can automate the process

File: api/3.12.3/docs/index.html (hunk 1 above)
Comment (fatihozer0) [reply api/3.12.3/docs/index.html:93]: Do I have to check every href? Sorry yo be that late. Since not in Jira I missed this PR

Comment (srknzl) [reply api/3.12.3/docs/index.html:93]: I think that's the case. You can exclude the links that start with `http`. However, you need to check all links in all readmes.

**Pull request closed without merging**